css/style.css

```diff
@@ -0,0 +1,66 @@
+body{
+    margin:0;
+    padding:0;
+}
+header{
+    margin:0;
+   padding:0;
+   text-align: center;
+   width:100%;
+}
+.bg-grey{
+    background: #2E2E2E;
+     padding:20px 0;
+}
+.bg-black{
+      background: rgb(0, 0, 0);
+}
+.bg-white{
+    background:rgb(246, 241, 241);
+}
+.container{
+    margin:auto;
+    display:flex;
+    width:1200px;
+}
+h1{
+    width:100%;
+}
+.col-50{
+    width:50%;
+}
+.col-50 img{
+    width:100%;
+    margin:0;
+}
+.col-50{
+  color:white;
+    font-size: xx-large; 
```

Comment:
`font-size: xx-large;`  is not servicing anything, and remove any other kind style in col-xx classes except **widths only**